docs/getting_help.rst

```diff
@@ -0,0 +1,35 @@
+.. _getting_help:
+
+Getting Help
+============
+
+Even if the best docs won't always answer your questions. If you can't
+figure out how to do what you want, there are a few places you can ask,
+depending on the nature of your question.
+
+* **Usage questions:** "How do I do X with OPS?", "Is there a way to do Y?"
+  "Why does this give an error?" Please ask at
+  http://mattermodeling.stackexchange.com, using the ``openpathsampling``
```

Comment:
```suggestion
  https://mattermodeling.stackexchange.com, using the ``openpathsampling``
```

Reply:
Also, how can we (as experienced users) sign up for email updates on these tags?